### ISSUE TITLE
🚨 [BUG] - asdf docs broken

### ISSUE THREAD

**Issue by d3sch41n** [facebookincubator/TTPForge#344]
### What happened?

The current docs for how to get started as a dev specify a series of steps based on asdf that don't work correctly. 

Consider the instructions to: 

```
source .asdf
```

When a new aspiring developer runs these instructions, it will actually kill their current shell session because this `source` call proceeds to make another `source` call under the hood which sources this external script:

https://raw.githubusercontent.com/l50/dotfiles/main/files/setup_asdf.sh 

That script assumes the existence of a global `.tool-versions` file, which the user will not have yet unless they have previously installed a global tool with `asdf`  (which our docs don't tell them to do, we just tell them to install asdf itself, and this should not be a prerequisite in any case) 

```
# Check for the global .tool-versions file employed by asdf
# to determine which versions of tools to install
if [[ ! -f "${HOME}/.tool-versions" ]]; then
    echo ".tool-versions file not found"
    exit 1
fi
```

The net result is that `exit 1` is called from within their current shell, rather than in a subprocess, which makes their current terminal tab instantly disappear (in iterm) or just say "process completed" (regular terminal).

**Comment by cedowens** [comment]
Reproduced the steps and had the same issue on my end:

<img width="279" alt="Screenshot 2023-09-19 at 9 28 07 AM" src="https://github.com/facebookincubator/TTPForge/assets/41086586/8fa45539-1ca9-407a-a4eb-2c78edac7655">

**Comment by l50** [comment]
@d3sch41n resolved with #345, closing
